supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/actors/GcpBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -73,8 +78,24 @@ import scala.util.control.NoStackTrace
 
 object GcpBatchAsyncBackendJobExecutionActor {
 
-  def StandardException(message: String, jobTag: String): Exception =
-    new Exception(s"Task $jobTag failed: $message")
+  private val VM_PREEMPTION_PATTERN = Pattern.compile(
+    "failed due to the following task event: \"Task state is updated from RUNNING to FAILED on zones/\\S+ due to Spot VM preemption with exit code 50001.\""
```

Comment:
Optional nice-to-have: document this message in the Batch section of RTD and maybe show how to locate it the operation details

Reply:
sure, could you point me to exactly where this would go?

Reply:
I was thinking [runtime attributes](https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/#preemptible), the primary place where we talk about preemptible.